src/engine/bpmcontrol.cpp

```diff
@@ -172,16 +183,19 @@ void BpmControl::slotFileBpmChanged(double file_bpm) {
 void BpmControl::slotAdjustBeatsFaster(double v) {
     BeatsPointer pBeats = m_pBeats;
     if (v > 0 && pBeats && (pBeats->getCapabilities() & Beats::BEATSCAP_SETBPM)) {
-        double new_bpm = math_min(200.0, pBeats->getBpm() + .01);
+        double new_bpm = pBeats->getBpm() + kBpmAdjustStep;
         pBeats->setBpm(new_bpm);
     }
 }
 
 void BpmControl::slotAdjustBeatsSlower(double v) {
     BeatsPointer pBeats = m_pBeats;
     if (v > 0 && pBeats && (pBeats->getCapabilities() & Beats::BEATSCAP_SETBPM)) {
-        double new_bpm = math_max(10.0, pBeats->getBpm() - .01);
-        pBeats->setBpm(new_bpm);
+        double bpm = pBeats->getBpm();
+        if (bpm > kBpmAdjustMin) {
```

Comment:
nit pick: if (bpm >= kBpmAdjustMin + kBpmAdjustStep) 
to avoid a conditional

Reply:
That's not correct if you want to ensure to reach kBpmAdjustMin eventually when repeatedly lowering the tempo.

Reply:
I've found an alternative and correct solution that is even more concise.